components/toolbar/style.scss

```diff
@@ -35,6 +35,10 @@ ul.components-toolbar {
 			height: $icon-button-size - 16px;
 		}
 	}
+
+	.block-editable__inline-toolbar & {
```

Comment:
Just wondering if we should move this to `blocks/editable/style.scss` instead? Should the generic toolbar component be aware of an editable wrapper.